service/service.go

```diff
@@ -63,18 +58,14 @@ const (
 
 // Application represents a collector application
 type Application struct {
```

Comment:
Can this change make any use cases for collector distros harder to implement? I've not worker with any recently so can't tell initial reaction is that hiding both service implementation and field on Application might restrict distros from implementing some sort of custom startup routine. Not sure if it is a real concern. Feel free to ignore if not.

Reply:
I am planning to expose the current service soon, this is just an intermediate step.